app/projectsmodel.h

```diff
@@ -26,7 +27,7 @@
  * and create list model from them. Available are full path to the file, name of the project
  * and short name of the project (clipped to N chars)
  */
-class ProjectModel : public QAbstractListModel
+class ProjectModel : public InputSearchModel
```

Comment:
if `InputSearchModel` is just container for one string value, why this value cannot be just added to ProjectModel? or does the inputSearchModel has any logic?

Reply:
There are two models - ProjectsModel and MerginProjectsModel. I wanted to have search string in both. However, MerginProjectsModel can be simplified and work without searchString